recipes/r-gam/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
+
```

Comment:
Why is that? Should R packages never include a bat file? This was generated with conda skeleton.